torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -4164,35 +4168,6 @@ def _get_default_cuda_device(module: nn.Module) -> torch.device:
     return torch.device("cuda", torch.cuda.current_device())
 
 
-def _free_storage(data: torch.Tensor) -> None:
-    """Free underlying storage of a Tensor."""
-    if data.storage().size() > 0:
-        # Since we're modifying the Tensor's Storage directly, make sure the Tensor
-        # is the sole occupant of the Storage.
-        assert (
-            data.storage_offset() == 0
-        ), "The tensor is not the sole occupant of the storage."
-        data.storage().resize_(0)  # type: ignore[attr-defined]
-
-
-@torch.no_grad()
-def _alloc_storage(data: torch.Tensor, size: torch.Size) -> None:
-    """Allocate storage for a tensor."""
-    if data.storage().size() == size.numel():  # no need to reallocate
-        return
-    assert (
-        data.storage().size() == 0
-    ), "Then tensor storage should have been resized to be 0."
-    data.storage().resize_(size.numel())  # type: ignore[attr-defined]
-
-def p_assert(cond: Any, s: Any) -> None:
```

Comment:
hm, I thought I added an argument to this function in one of my recent PRs, but not sure if this is rebased

Reply:
The added argument is there now after rebasing.